### PR TITLE
Add --no-backup option to databse:copy and database:copy:all

### LINKED ISSUE #124: Option to disable backups when moving DBs
**Is your feature request related to a problem? Please describe.**
I want to use the cli to copy databases from a multisite with >30 databases. By default `database:copy` and `database:copy:all` backup the DBs on the target environment first, which could create a lot of clutter on the server. I've had issue with running out of disk space as well.

**Describe the solution you'd like**
Add the `--no-backup` option to `database:copy` and `database:copy:all` so you can skip the copy operation.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by earthday47**
Resolves https://github.com/typhonius/acquia_cli/issues/124

### CODE REVIEW

**Comment by typhonius** [comment]
Good idea - thanks for your contribution!

Couple of small comments.

- Please add tests (I've written tests for most of the other methods so you should be able to add another item to https://github.com/typhonius/acquia_cli/blob/master/tests/Commands/DbCommandTest.php to get expected output. You can run the tests locally with `composer test`
- Can we also alter the `backupAndMoveDbs` method to use your new `moveDbs` method so we're doing all db moving in one place.
- I've got `no-backup` functionality in the code deploy task.  https://github.com/typhonius/acquia_cli/blob/75a5126d0d32b7a746f60afd508d970f87b6c590/src/Commands/CodeCommand.php#L89 so I think you could alter your commit to use `if (!$options['no-backup']) {` for consistency.
- Couple of minor phpcs fixes that have been identified by CI.

```
FILE: ...ome/travis/build/typhonius/acquia_cli/src/Commands/DbCommand.php
----------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
----------------------------------------------------------------------
 129 | ERROR | [x] Expected 1 space after closing brace; newline
     |       |     found
 163 | ERROR | [x] Expected 1 space after closing brace; newline
     |       |     found
----------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```

**Comment by typhonius** [comment]
I've also just switched CI over to GitHub Actions so if you rebase your branch against master you should get some improved code reviews/checks.

**Comment by earthday47** [comment]
hi @typhonius thank you! All of your feedback sounds good to me, I'll get to work on them

**Comment by earthday47** [comment]
Updated with tests... it looks like my force push cancelled the GitHub checks, but `composer test` is clean on my local.

**Comment by typhonius** [comment]
Thanks, I’ve seen that error on other PRs and looks like it’s an unrelated step later where we rebuild the phar tool. I’ll push an update later on today because I have an inkling about how to fix it.

File: src/Commands/DbCommand.php [hunk 1]
```diff
@@ -151,7 +166,11 @@ public function dbCopyAll($uuid, $environmentFrom, $environmentTo)
                 )
             )
         ) {
-            $this->backupAndMoveDbs($uuid, $environmentFrom, $environmentTo);
+            if (!$options['no-backup']) {
+                $this->moveDbs($uuid, $environmentFrom, $environmentTo);
+            } else {
+                $this->moveDbs($uuid, $environmentFrom, $environmentTo, null, $backup = false);
```
Comment (typhonius) [inline src/Commands/DbCommand.php]: Switch `$backup = false` to just `false`.

File: src/Commands/DbCommand.php [hunk 2]
```diff
@@ -123,7 +129,11 @@ public function dbCopy($uuid, $environmentFrom, $environmentTo, $dbName)
                 )
             )
         ) {
-            $this->backupAndMoveDbs($uuid, $environmentFrom, $environmentTo, $dbName);
+            if (!$options['no-backup']) {
+                $this->moveDbs($uuid, $environmentFrom, $environmentTo, $dbName);
+            } else {
+                $this->moveDbs($uuid, $environmentFrom, $environmentTo, $dbName, $backup = false);
```
Comment (typhonius) [inline src/Commands/DbCommand.php]: Switch $backup = false to just false.

**Review by typhonius — COMMENTED** [review]
If you are able to rebase (again) against master and fix the following two minor changes then I'll merge it :)

**Comment by earthday47** [comment]
All green! 😁
Thanks for your attention and quick review of this request - we already use the CLI as a crucial part of our workflow and this will only help improve it more.

**Pull request merged**